<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Веб-Браузер</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ссылка : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leUrl"/>
      </item>
      <item>
       <widget class="QPushButton" name="pbGo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Перейти</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWebView" name="webView">
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>529</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_Browser"/>
   <addaction name="act_Table"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_Home"/>
   <addaction name="act_Back"/>
   <addaction name="act_Next"/>
   <addaction name="act_Add"/>
   <addaction name="act_Refresh"/>
   <addaction name="act_Del"/>
  </widget>
  <action name="act_Browser">
   <property name="icon">
    <iconset>
     <normaloff>../../pics/Website-icon.png</normaloff>../../pics/Website-icon.png</iconset>
   </property>
   <property name="text">
    <string>Показать браузер</string>
   </property>
   <property name="toolTip">
    <string>Показать браузер</string>
   </property>
  </action>
  <action name="act_Table">
   <property name="icon">
    <iconset>
     <normaloff>../../pics/web-page.png</normaloff>../../pics/web-page.png</iconset>
   </property>
   <property name="text">
    <string>Таблица закладок</string>
   </property>
   <property name="toolTip">
    <string>Таблица закладок</string>
   </property>
  </action>
  <action name="act_Home">
   <property name="icon">
    <iconset>
     <normaloff>../../pics/home.png</normaloff>../../pics/home.png</iconset>
   </property>
   <property name="text">
    <string>Домашняя страница</string>
   </property>
   <property name="toolTip">
    <string>Домашняя страница</string>
   </property>
  </action>
  <action name="act_Add">
   <property name="icon">
    <iconset>
     <normaloff>../../pics/Add.png</normaloff>../../pics/Add.png</iconset>
   </property>
   <property name="text">
    <string>Добавить в таблицу</string>
   </property>
   <property name="toolTip">
    <string>Добавить в таблицу</string>
   </property>
  </action>
  <action name="act_Del">
   <property name="icon">
    <iconset>
     <normaloff>../../pics/Del.png</normaloff>../../pics/Del.png</iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="toolTip">
    <string>Удалить</string>
   </property>
  </action>
  <action name="act_Next">
   <property name="icon">
    <iconset>
     <normaloff>../../pics/go.ico</normaloff>../../pics/go.ico</iconset>
   </property>
   <property name="text">
    <string>Вперед</string>
   </property>
   <property name="toolTip">
    <string>Вперед</string>
   </property>
  </action>
  <action name="act_Back">
   <property name="icon">
    <iconset>
     <normaloff>../../pics/back.png</normaloff>../../pics/back.png</iconset>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
   <property name="toolTip">
    <string>Назад</string>
   </property>
  </action>
  <action name="act_Refresh">
   <property name="icon">
    <iconset>
     <normaloff>../../pics/refresh.png</normaloff>../../pics/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
   <property name="toolTip">
    <string>Обновить</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
